common/app/model/content/Atoms.scala

```diff
@@ -189,3 +201,17 @@ object Quiz extends common.Logging {
     )
   }
 }
+
+object InteractiveAtom {
+  def make(id: String, interactive: atomapi.interactive.InteractiveAtom): InteractiveAtom = {
```

Comment:
Happy to do so, but should I change the other atom `make` methods to `apply` too for consistency?